osu.Game/Overlays/Profile/Header/Components/DailyChallengeStreakDisplay.cs

```diff
@@ -0,0 +1,111 @@
+            InternalChildren = new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colourProvider.Background4,
+                },
+                new FillFlowContainer
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Padding = new MarginPadding(5f),
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Horizontal,
+                    Children = new Drawable[]
+                    {
+                        new OsuTextFlowContainer(s => s.Font = OsuFont.GetFont(size: 12))
+                        {
+                            AutoSizeAxes = Axes.Both,
+                            // Text = UsersStrings.ShowDailyChallengeTitle
+                            Text = "Daily\nChallenge",
+                            Margin = new MarginPadding { Horizontal = 5f, Bottom = 2f },
+                        },
+                        new Container
+                        {
+                            AutoSizeAxes = Axes.X,
+                            RelativeSizeAxes = Axes.Y,
+                            CornerRadius = 5f,
+                            Masking = true,
+                            Children = new Drawable[]
+                            {
+                                new Box
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Colour = colourProvider.Background6,
+                                },
+                                dailyStreak = new OsuSpriteText
+                                {
+                                    Anchor = Anchor.Centre,
+                                    Origin = Anchor.Centre,
+                                    UseFullGlyphHeight = false,
+                                    Colour = colourProvider.Content2,
+                                    Margin = new MarginPadding { Horizontal = 10f, Vertical = 5f },
+                                },
+                            }
+                        },
+                    }
+                },
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            User.BindValueChanged(_ => updateDisplay(), true);
+        }
+
+        private void updateDisplay()
+        {
+            if (User.Value == null)
+            {
+                dailyStreak.Text = "-";
+                return;
+            }
+
+            var statistics = User.Value.User.DailyChallengeStatistics;
+            // dailyStreak.Text = UsersStrings.ShowDailyChallengeUnitDay(statistics.DailyStreakCurrent);
+            dailyStreak.Text = $"{statistics.DailyStreakCurrent}d";
+            dailyStreak.Colour = colours.ForRankingTier(DailyChallengeStreakTooltip.TierForDaily(statistics.DailyStreakCurrent));
```

Comment:
This is the incorrect value and it shouldn't have color. You can see differences with peppy's profile.

Reply:
Yeah I noticed, was shocked about it. I'll change to match web I guess.

Reply:
Forgot to change to `statistics.PlayCount` and probably rename field.

Reply:
Wait what...

Reply:
TIL the display doesn't show the streak but show what's supposedly the number of times the user plays a daily challenge

Reply:
Playcount seems to have color now: https://github.com/ppy/osu-web/pull/11381

Reply:
I've updated for this.